File: GestionInterim/app/src/main/java/com/example/gestioninterim/MainActivity.java
package com.example.gestioninterim;

import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.FragmentManager;
import androidx.fragment.app.FragmentTransaction;

import android.os.Bundle;

import com.example.gestioninterim.slides.FragmentSlideParent;

public class MainActivity extends AppCompatActivity {

    FragmentManager fragmentManager;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        fragmentManager = getSupportFragmentManager();

        FragmentTransaction fragmentTransaction = fragmentManager.beginTransaction();

        FragmentSlideParent fragment = new FragmentSlideParent();

        fragmentTransaction.add(R.id.containerFirstSlides, fragment);

        fragmentTransaction.commit();

    }
}

File: GestionInterim/app/src/main/java/com/example/gestioninterim/MainActivity.kt
package com.example.gestioninterim

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.fragment.app.FragmentManager
import com.example.gestioninterim.slides.FragmentSlideParent

class MainActivity : AppCompatActivity() {
    var fragmentManager: FragmentManager? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        fragmentManager = supportFragmentManager
        val fragmentTransaction = fragmentManager!!.beginTransaction()
        val fragment = FragmentSlideParent()
        fragmentTransaction.add(R.id.containerFirstSlides, fragment)
        fragmentTransaction.commit()
    }
}